frontend/src/components/organism/Projections.tsx

```diff
@@ -0,0 +1,56 @@
+import moment from 'moment';
+import React from 'react';
+import styled from 'styled-components';
+import { useTransactionState } from '../../store/contexts/transactions';
+
+const Projections: React.FC<{ className?: string }> = ({ className }) => {
+  const transactionState = useTransactionState();
+  const transactions = transactionState.transactions;
+  const currentBalance = 0;
+  let accumulatedBalance = currentBalance;
+  const renderTransactions = () =>
+    transactions
+      .filter(t =>
+        moment(t.date).isBetween(
+          moment().startOf('month'),
+          moment().add(5, 'years')
+        )
+      )
+      .sort((t1, t2) => (t2.date > t1.date ? -1 : 1))
+      .map(t => {
+        t.type === 'IN'
```

Comment:
I think it would be cleaner to write

```typescript
accumulatedBalance += (t.type === 'IN' ? t.money : -t.money) / 100;
```

The reason is that performing assignments within a ternary expression generally is disencouraged.